static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -145,28 +145,30 @@ define([
         var $savers = bonzo(qwery(this.classes.saveThisArticle));
 
         $savers.each(function (saver) {
-            var $saver = bonzo(saver);
-            $saver.css('display', 'block');
-            var templateData = {
-                icon: bookmarkSvg,
-                isSaved: options.isSaved,
-                position: $saver.attr('data-position'),
-                config: config
-            };
-            if (options.url) {
-                $saver.html(template(saveLink,
-                    assign({ url: options.url }, templateData))
-                );
-            } else {
-                $saver.html(template(saveButton, templateData));
-
-                bean.one($saver[0], 'click', this.classes.saveThisArticleButton,
-                    this[options.isSaved ? 'deleteArticle' : 'saveArticle'].bind(this,
-                        config.page.pageId,
-                        shortUrl
-                    )
-                );
-            }
+             var $saver = bonzo(saver);
+             var templateData = {
+                 icon: bookmarkSvg,
+                 isSaved: options.isSaved,
+                 position: $saver.attr('data-position'),
+                 config: config
+             };
+            fastdom.write(function(){
+                $saver.css('display', 'block');
+                if (options.url) {
+                    $saver.html(template(saveLink,
+                        assign({ url: options.url }, templateData))
+                    );
+                } else {
+                    $saver.html(template(saveButton, templateData));
+
+                    bean.one($saver[0], 'click', this.classes.saveThisArticleButton,
```

Comment:
You lost your scope: https://sentry.io/the-guardian/client-side-prod/issues/165900906/